pkg/policies/engine/rego/boilerplate.rego.tmpl

```diff
@@ -0,0 +1,22 @@
+{{if .NeedsResult}}result := {
+    "skipped": skipped,
+    "violations": violations,
+    "skip_reason": skip_reason,
+}
+
+{{end}}{{if .NeedsSkipReason}}default skip_reason := ""
```

Comment:
I think you can use trim markers to avoid this weird end-if in the same line and to remove those blanks. See https://pkg.go.dev/text/template#hdr-Text_and_spaces:
```suggestion
{{end}}
{{- if .NeedsSkipReason -}} 
default skip_reason := ""
```